docs/updates/development-update-24.07.md

```diff
@@ -0,0 +1,102 @@
+# Development Update 24.07
+
+## Overview
+
+This is an update (code drop) to incorporate changes made to support
+the Intel IPU E2100.
+
+## Breaking Changes
+
+### Client Flags
+
+- The Stratum `CredentialsManager` class and `gnmi_cli` tool have been aligned
+  with the current upstream version, and IPDK modifications have been
+  overhauled to be more consistent with Stratum as a whole.
+
+  The `-grpc_client_cert_req_type` flag is now `-client_cert_req_type`, and
+  its value names have been shortened.
+
+  | Old Name | New Name |
+  | -------- | -------- |
+  | NO_REQUEST_CLIENT_CERT | NO_REQUEST |
```

Comment:
Should we update the new string names to lowecase based on the open PR?

Reply:
Done.